src/components/CoursesTabs.js

```diff
@@ -75,38 +79,47 @@ const useStyles = makeStyles(() => ({
   },
 }))
 
-const OffCampus = ({ classes }) => {
+const OffCampus = ({ classes, set_campus_visible }) => {
   return (
     <Card className={classes.infoCard}>
       <CardHeader
         className={classes.header}
         classes={{
           title: classes.headerContent,
+          action: classes.action
```

Comment:
you're adding the `action` style, but I don't see it defined anywhere. I believe that in my branch, that style is making sure that the icons are centered vertically, and I would suggest the same styling for each of these cards, too

Reply:
Just sent the updates.